app/connector/kafka/src/main/java/io/syndesis/connector/kafka/KafkaMetaDataRetrieval.java

```diff
@@ -47,15 +47,15 @@
     private static final Logger LOG = LoggerFactory.getLogger(KafkaMetaDataRetrieval.class);
 
     static final CustomResourceDefinition KAFKA_CRD = new CustomResourceDefinitionBuilder()
-        .withApiVersion("apiextensions.k8s.io/v1beta1")
+        .withApiVersion("apiextensions.k8s.io/v1beta2")
         .withKind("CustomResourceDefinition")
         .withNewMetadata()
             .withName("kafkas.kafka.strimzi.io")
         .endMetadata()
         .withNewSpec()
             .withGroup("kafka.strimzi.io")
             .withScope("Namespaced")
-            .withVersion("v1beta1")
+            .withVersion("v1beta2")
```

Comment:
Should this be part of this PR?

Reply:
I will submit another PR with this change.